src/java-runtime/java-runtime.targets

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<ItemGroup>
+  <_RuntimeOutput Include="$(OutputPath)java_runtime.jar">
+    <OutputJar>$(OutputPath)java_runtime.jar</OutputJar>
+    <OutputDex>$(OutputPath)java_runtime.dex</OutputDex>
+    <IntermediateRuntimeOutputPath>$(IntermediateOutputPath)release</IntermediateRuntimeOutputPath>
+    <IntermediateRuntimeClassesTxt>$(IntermediateOutputPath)release.txt</IntermediateRuntimeClassesTxt>
+    <RemoveItems>java\mono\android\debug\MultiDexLoader.java</RemoveItems>
+    <RemoveItems>java\mono\android\MonkeyPatcher.java</RemoveItems>
+    <RemoveItems>java\mono\android\ResourcePatcher.java</RemoveItems>
+    <RemoveItems>java\mono\android\Seppuku.java</RemoveItems>
+  </_RuntimeOutput>
+  <_RuntimeOutput Include="$(OutputPath)java_runtime_fastdev.jar">
+    <OutputJar>$(OutputPath)java_runtime_fastdev.jar</OutputJar>
+    <OutputDex>$(OutputPath)java_runtime_fastdev.dex</OutputDex>
+    <IntermediateRuntimeOutputPath>$(IntermediateOutputPath)fastdev</IntermediateRuntimeOutputPath>
+    <IntermediateRuntimeClassesTxt>$(IntermediateOutputPath)fastdev.txt</IntermediateRuntimeClassesTxt>
+    <RemoveItems>java\mono\android\release\MultiDexLoader.java</RemoveItems>
+  </_RuntimeOutput>
+</ItemGroup>
+<Target Name="Build" 
+      Inputs="@(AllRuntimeSource)"
+      Outputs="@(_RuntimeOutput)"
+>
+  <MakeDir Directories="%(_RuntimeOutput.IntermediateRuntimeOutputPath)" />
+  <MakeDir Directories="$(OutputPath)" />
+  <ItemGroup>
+    <_RuntimeSource Include="@(AllRuntimeSource)" />
+    <_RuntimeSource Remove="%(_RuntimeOutput.RemoveItems)" />
+  </ItemGroup>
+  <WriteLinesToFile
+      File="%(_RuntimeOutput.IntermediateRuntimeClassesTxt)"
+      Lines="@(_RuntimeSource)"
+      Overwrite="True"
+  />
+  <PropertyGroup>
+    <_Target Condition="'$(JavacSourceVersion)' != ''">-source $(JavacSourceVersion) -target $(JavacSourceVersion)</_Target>
+    <_AndroidJar>"$(AndroidToolchainDirectory)\sdk\platforms\android-$(AndroidFirstPlatformId)\android.jar"</_AndroidJar>
```

Comment:
Making sure I follow, this should be the `android.jar` for the latest stable API level: https://github.com/xamarin/xamarin-android/pull/2536#discussion_r262931625

And we wouldn't need `$(AndroidFirstApiLevel )` or `$(AndroidFirstPlatformId)` anymore in `Configuration.props`.

Reply:
@jonathanpeppers not it needs to be against our minimum supported api, not the latest. Otherwise it won't work on older devices.